github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-postsubmits.yaml

```diff
@@ -37,3 +37,101 @@ postsubmits:
         resources:
           requests:
             memory: "8Gi"
+  - name: push-release-kubevirt-tag
+    branches:
+    - release-\d+\.\d+
+    cluster: ibm-prow-jobs
+    always_run: true
+    optional: false
```

Comment:
does this job execute as a result of a tag getting created or as a result of every commit to a release branch?

Reply:
i see, it executes on every commit and the logic in kubevirt only does something if the commit points to a tag

Reply:
Exactly.